### DISCUSSION TITLE
How do you serve static file on "/"?

### DISCUSSION THREAD

**Discussion by edyu** [OxygenFramework/Oxygen.jl#99] [Q&A] [answered]
I tried all of the following but none of them allows me to serve static files under "/" directly.
`@staticfiles "content" ""`
`@staticfiles "content"`
`@staticfiles "content" "/"`

**Comment by ndortega** [comment]
Unfortunately there's no support for that feature. Funny enough, that same issue was brought up yesterday day in the discord server.

This issue will be moved to the top of my priority list since I've seen it mentioned several times already

**Reply by edyu** [reply]
That was me in the discord. :)

**Comment by ndortega** [comment, marked as answer]
This feature is now supported in the latest version v1.1.7. I've also added corresponding functions (named the same) to mount files. So now you can mount them like this: `staticfiles("content", "/")`.
